src/main/java/games/strategy/triplea/ui/MovePanel.java

```diff
@@ -1265,7 +1265,7 @@ public void mouseMoved(final Territory territory, final MouseDetails me) {
             // is the only one for
             // which the route may actually change much)
             if (unitsThatCanMoveOnRoute.size() < selectedUnits.size() && (unitsThatCanMoveOnRoute.size() == 0
-                || Match.allMatchNotEmpty(unitsThatCanMoveOnRoute, Matches.UnitIsAir))) {
+                || (!unitsThatCanMoveOnRoute.isEmpty() && Match.allMatch(unitsThatCanMoveOnRoute, Matches.UnitIsAir)))) {
```

Comment:
Sorry, I didn't see this after 541a4e3, and it broke the build...  Checkstyle: "Line is longer than 120 characters (found 121)."